identity-wallet/src/state/reducers/dev_mode.rs

```diff
@@ -38,20 +42,217 @@ lazy_static! {
     );
 }
 
-pub async fn set_dev_mode(state: AppState, action: Action) -> Result<AppState, AppError> {
-    if let Some(enabled) = listen::<SetDevMode>(action).map(|payload| payload.enabled) {
-        return Ok(AppState {
-            dev_mode_enabled: enabled,
-            current_user_prompt: None,
-            ..state
-        });
+const PROFILE_PW: &str = "sup3rSecr3t";
+
+pub async fn load_dev_profile(state: AppState, action: Action) -> Result<AppState, AppError> {
+    info!("Load dev profile: {:?}", action);
+
+    if let Some(dev_profile) = listen::<DevProfile>(action) {
+        match dev_profile.profile {
+            ProfileType::Ferris => return load_ferris_profile().await,
+            ProfileType::Dragon => return load_dragon_profile(state).await,
+            ProfileType::None => {}
+        }
+    }
+
+    Ok(state)
+}
+
+pub async fn toggle_dev_settings(mut state: AppState, _action: Action) -> Result<AppState, AppError> {
+    info!("Toggle dev settings");
+
+    if state.dev_profile.is_some() {
+        state.dev_profile = None;
+    } else {
+        state.dev_profile = Some(ProfileType::None);
     }
+
+    state.current_user_prompt = None;
+
     Ok(state)
 }
 
-pub async fn load_dev_profile(_state: AppState, _action: Action) -> Result<AppState, AppError> {
-    info!("load dev profile");
+pub async fn login_profile(state: AppState) -> Result<AppState, AppError> {
+    command::reduce(
+        state,
+        Arc::new(UnlockStorage {
+            password: PROFILE_PW.to_string(),
+        }),
+    )
+    .await
+}
+
+async fn create_new_profile(state: AppState) -> Result<AppState, AppError> {
+    let create_new = CreateNew {
+        name: "Dragon".to_string(),
+        picture: "🐲".to_string(),
```

Comment:
Can you replace the actual icon with `"&#x1F432"` which is the [codepoint](https://emojipedia.org/dragon-face#technical) for dragon-face?

Reply:
fixed